pkg/client/clientset/versioned/clientset.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by client-gen. DO NOT EDIT.
+
+package versioned
+
+import (
+	loggingv1alpha1 "github.com/platform9/fluentd-operator/pkg/client/clientset/versioned/typed/logging/v1alpha1"
+	discovery "k8s.io/client-go/discovery"
+	rest "k8s.io/client-go/rest"
+	flowcontrol "k8s.io/client-go/util/flowcontrol"
+)
+
+type Interface interface {
+	Discovery() discovery.DiscoveryInterface
+	LoggingV1alpha1() loggingv1alpha1.LoggingV1alpha1Interface
+	// Deprecated: please explicitly pick a version if possible.
+	Logging() loggingv1alpha1.LoggingV1alpha1Interface
+}
+
+// Clientset contains the clients for groups. Each group has exactly one
+// version included in a Clientset.
+type Clientset struct {
+	*discovery.DiscoveryClient
+	loggingV1alpha1 *loggingv1alpha1.LoggingV1alpha1Client
+}
+
+// LoggingV1alpha1 retrieves the LoggingV1alpha1Client
+func (c *Clientset) LoggingV1alpha1() loggingv1alpha1.LoggingV1alpha1Interface {
+	return c.loggingV1alpha1
+}
+
+// Deprecated: Logging retrieves the default version of LoggingClient.
```

Comment:
why is this depreciated? It's strange to see such language in new code we are adding.